core-services/image-mirroring/openshift-kni/mapping_openshift-kni_quay

```diff
@@ -1,8 +1,10 @@
 # numaresources-operator
-registry.ci.openshift.org/ocp-kni/numaresources-operator:numaresources-operator-tests quay.io/openshift-kni/numaresources-operator-tests:4.18.999-snapshot
```

Comment:
Why are we dropping this one?

Reply:
we are moving them to 4.19, we keep publishing the bundles. I guess I can reorder the lines to make it easier to read the diff

Reply:
on second look the ordering seems good:
- tests (only from main)
- bundle (main, then releases from newest to oldest)
- operator (main, then releases from newest to oldest)
- rte (CI only)
- must-gather (CI only)